pkg/kontainer-engine/drivers/aks/aks_driver.go

```diff
@@ -923,9 +923,8 @@ func (state state) hasHTTPApplicationRoutingSupport() bool {
 
 func (d *Driver) ensureLogAnalyticsWorkspaceForMonitoring(ctx context.Context, client *operationalinsights.WorkspacesClient, state state) (workspaceID string, err error) {
 	// Please keep in sync with
-	// https://github.com/Azure/azure-cli/blob/release/src/azure-cli/azure/cli/command_modules/acs/custom.py#L1996
-
-	locationToOmsRegionCodeMap := map[string]string{
+	//https://github.com/Azure/azure-cli/blob/release/src/azure-cli/azure/cli/command_modules/acs/custom.py#L3091
```

Comment:
Should remove line number since it appears to be wrong and probably changes often enough.